app/views/katello/api/v2/packages/show.json.rabl

```diff
@@ -1,11 +1,4 @@
 object @resource
 
 extends "katello/api/v2/packages/base"
-
-attributes :description
-attributes :license, :buildhost, :vendor, :relativepath
-attributes :children, :checksumtype, :size, :url, :build_time, :summary, :group, :requires, :provides, :files
-
-node :human_readable_size do |package|
-  number_to_human_size(package.size)
-end
+extends "katello/api/v2/packages/backend", :object => Katello::Pulp::Rpm.new(@resource.uuid)
```

Comment:
Did you consider potentially doing at the top of, say the RPM backend rabl:

```
object Katello::Pulp::Rpm.new(@resource.uuid)
``` 

I know the current method keeps the backend rabl agnostic, I'm just not sure I ever see it being used without doing the above transformation? Just a thought, as I am 50/50 on each idea.